src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/ExpectedOutputs/OneRunWithRules.sarif

```diff
@@ -45,18 +47,19 @@
               "some_key": "FoxForceFive"
             }
           },
-          "C2002": {
+          {
             "id": "C2002",
             "fullDescription": {
               "text": "Catfish season continuous hen lamb include dose copy grant."
             },
             "configuration": {
               "enabled": true,
-              "defaultLevel": "error"
+              "defaultLevel": "error",
+              "defaultRank": 0.0
```

Comment:
>defaultRank [](start = 15, length = 11)

I do not know why the serializer is emitting `defaultRank` with its default value, but I wouldn't hold this PR to look at it. Considering this occurs in the v1->v2 transform, I don't think we should _ever_ take the time to look at it. If something similar shows up in direct production, that will be the time to look at it.